src/modules/tools/zprobe/ZGridStrategy.cpp

```diff
@@ -44,6 +44,10 @@
 
        leveling-strategy.ZGrid-leveling.wait_for_probe  true
 
+    The machine can be told to home in one of the following modes:
+
+       leveling-strategy.ZGrid-leveling.home_before_probe  2;    # 0: NOHOME  1: HOMEXY  2: HOMEXYZ (Default HOMEXYZ)
```

Comment:
fixing that up right now


On Thu, Jul 2, 2015 at 9:43 PM Jim Morris <notifications@github.com> wrote:

> In src/modules/tools/zprobe/ZGridStrategy.cpp
> <https://github.com/Smoothieware/Smoothieware/pull/681#discussion_r33815147>
> :
>
> > @@ -44,6 +44,10 @@
> >
> >         leveling-strategy.ZGrid-leveling.wait_for_probe  true
> >
> > +    The machine can be told to home in one of the following modes:
> > +
> > +       leveling-strategy.ZGrid-leveling.home_before_probe  2;    # 0: NOHOME  1: HOMEXY  2: HOMEXYZ (Default HOMEXYZ)
>
> I'm nitpicking here but why use 0, 1, 2 and not nohome, homexy, hmexyz in
> the config, it is much clearer, and I prefer that to cryptic numbers in the
> config.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Smoothieware/Smoothieware/pull/681/files#r33815147>.
>